dj_database_url.py

```diff
@@ -196,7 +191,4 @@ def parse(
     if options:
         parsed_config["OPTIONS"] = options
 
-    if engine:
-        parsed_config["ENGINE"] = engine
```

Comment:
`engine` is always Truthy as in the None case we parse it earlier on, so I've just set this directly in the earlier `update` call.